backend/inspirehep/search/facets.py

```diff
@@ -298,17 +308,19 @@ def citations_by_year():
         "citeable",
         "refereed",
         "citation_count",
+        "citation_count_without_self_citations",
         "author_count",
         "earliest_date",
         "collaboration",
     ]
     filters = get_filters_without_excluded(hep_filters(), excluded_filters)
-    map_script = """
-        def years = params._source.citations_by_year != null ? params._source.citations_by_year : [];
-        for (element in years) {
-            state.merge(element.year.toString(), element.count, (x, y) -> x + y)
-        }
-    """
+
+    map_script = (
+        f"def years = params._source.citations_by_year != null ? params._source.citations_by_year : [];"
+        "for (element in years) {"
+        "    state.merge(element.year.toString(), element.count, (x, y) -> x + y)"
+        "}"
+    )
```

Comment:
you can revert this part, now you don't need to interpolate stuff

Reply:
:+1: